### PR TITLE
Change Arch setup

### PR DESCRIPTION
**Pull request by jmta**
Found a few issues with the arch install recently. Certain things could only hold true for my arch install so could do with some additional testers.

Sorry if making a pull request wasn't the right thing to do, first time contributing to a git project.

### CODE REVIEW

**Comment by mhkabir** [comment]
@eyeam3 @julianoes could you please verify?

File: en/setup/dev_env_linux_boutique.md [hunk 1]
```diff
@@ -95,8 +95,10 @@ sudo yum install glibc.i686 ncurses-libs.i686
 
 <div class="host-code"></div>
 
+Ensure you have the multilib repository enabled.
+
 ```sh
-sudo pacman -S base-devel lib32-glibc git-core python-pyserial zip python-empy
+sudo pacman -S base-devel lib32-glibc git-core python-pyserial zip
```
Comment (jmta) [inline en/setup/dev_env_linux_boutique.md:101]: python-empy not found in my case. May be part of multilib, this issue was fixed before adding multilib repo

File: en/setup/dev_env_linux_boutique.md [hunk 2]
```diff
@@ -136,8 +138,7 @@ cd ~
 wget https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q3-update/+download/gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2
 tar -jxf gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2
 exportline="export PATH=$HOME/gcc-arm-none-eabi-4_9-2015q3/bin:\$PATH"
-if grep -Fxq "$exportline" ~/.profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-. ~/.profile
+if grep -Fxq "$exportline" ~/.bash_profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
```
Comment (jmta) [inline en/setup/dev_env_linux_boutique.md:141]: No ~/.profile in my case, could be just my setup.

File: en/setup/dev_env_linux_boutique.md [hunk 3]
```diff
@@ -149,11 +150,12 @@ cd ~
 wget https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
 tar -jxf gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
 exportline="export PATH=$HOME/gcc-arm-none-eabi-5_4-2016q2/bin:\$PATH"
-if grep -Fxq "$exportline" ~/.profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-. ~/.profile
+if grep -Fxq "$exportline" ~/.bash_profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
 popd
 ```
 
+Now restart your machine.
```
Comment (jmta) [inline en/setup/dev_env_linux_boutique.md:157]: Restart needed as to reload the ~/.bash_profile

**Review by LorenzMeier — APPROVED** [review]

**Pull request merged**